### ISSUE TITLE
Found conflicts between different versions of the same dependent assembly.

### ISSUE THREAD

**Issue by balexandr** [andrewdavey/postal#32]
When installing the MVC postal nuGet package I get a warning during the build process.

"Found conflicts between different versions of the same dependent assembly."

I tried searching the internets and messing with the web.config. It got to the ponit where even when I uninstalled the package, I was still getting the issue. I had create a brand new project.

I think it has something to do with the clash or Razor engines?

**Comment by petejstephens** [comment]
Since installing version 7.3 of Postal onto an MVC4 project deployable to Azure, i've discovered that the RazorEngine used in this version of Postal uses system.web.razor 1.0.0.0, unfortunately MVC4 uses version 2.0.0.0  of system.web.razor. To overcome this I set a bindingRedirect of system.web.razor to version 2.0.0.0, which is supposed to allow 3rd party assemblies to use different versions of sub-assemblies, this works fine locally but when I deploy this to Azure I then get, 

"Could not load type 'System.Web.Razor.Parser.SyntaxTree.CodeSpan' from assembly 'System.Web.Razor, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.", 

Which implies to me that the Razor engine is using something that has not been deprecated in version 2.0.0.0 of system.web.razor......

So I then tried to deploy my system with system.web.razor 1.0.0.0 instead, but I quite predictably get an incompatible references warnings. 

So now I'm a bit stuck, I really would like to fix this as my currently deployed web app soundtracktoyour.com depends heavily on postal and is now become very unstable since I've moved to MVC4. Which I believe a move to version 7.03 of postal will fix.... Or is it the case I've jumped ahead of myself and assumed that postal 7 is compatible with MVC4, when in fact RazorEngine which it uses is not....

Help, as I've run out of ideas....
